idaes/models_extra/power_generation/unit_models/helm/compressor.py

```diff
@@ -158,7 +158,7 @@ def initialize_build(
         solve_log = idaeslog.getSolveLogger(self.name, outlvl, tag="unit")
 
         # Create solver
-        slvr = get_solver(solver, optarg)
+        opt = get_solver(solver, optarg)
```

Comment:
I actually did a find-and-replace on `slvr` to change it to `opt` in my project. I didn't see that the search scope wasn't limited at first, so this got changed too. I can change it back.

I preferred `opt` because I thought having both `solver` and `slvr` might be confusing. If we're doing `slvr`, it can get changed back.

Reply:
I tend to use `solver_obj` in these cases to make it clear what it is. I agree that also having the `solver` argument makes this a bit confusing, but `opt` does not tell a user what this object is and also implied "optimization", whereas in this case we are solving a square problem.